### DISCUSSION TITLE
Need help to get our website social.britehoods.com

### DISCUSSION THREAD

**Discussion by vedlettuce** [mastodon/mastodon#29897] [Q&A] [answered]
Hi,

Firstly, great work, and kudos to the team that built this awesome product. I bumped on Mastodon accidentally and found it to be the right platform for our users who would like to discuss & share about sustainability, climate change etc. We've installed it on our own servers, and everything works fine, but between our desire and reality, our website https://social.britehoods.com is stuck at verification, which is taking way too long I feel. Can anyone help me with the verification, please.

Thanks in advance.

**Comment by renchap** [comment]
What do you mean by "stuck at verification"?

**Comment by vedlettuce** [comment]
I am seeing this message since quiet sometime, "Your application is pending review by our staff. This may take some time. You will receive an e-mail if your application is approved. "

**Reply by renchap** [reply]
This is a bug, and is documented[ in the 4.2.8 release notes](https://github.com/mastodon/mastodon/releases/tag/v4.2.8) (under Known Issues)

You need to manually approve this first account using `RAILS_ENV=production bin/tootctl modify <username> --approved` on the command line

**Comment by vedlettuce** [comment]
Thank you all for helping,I tried executing the command  `RAILS_ENV=production bin/tootctl modify <username> --approved` it returned me `Could not find command "modify"`
Then I followed the release notes and installed v4.2.8 to see if it could resolve the above error but it again failed with the same message

Did I miss any steps?

**Reply by StrangeMinds1960** [reply, marked as answer]
That should be `RAILS_ENV=production bin/tootctl accounts modify <username> --approved`

and should be done as the mastodon user from the /home/mastodon/live directory

https://docs.joinmastodon.org/admin/tootctl/#accounts-modify

**Reply by renchap** [reply]
This is correct, the release notes were wrong, I fixed them, sorry for this!

**Reply by vedlettuce** [reply]
@StrangeMinds1960 just wanted to update you that the correct command is `RAILS_ENV=production bin/tootctl accounts modify <username> --approve`

The documentation helped me, hope this helps others in the future too.

Thank you all!

**Reply by StrangeMinds1960** [reply]
You're right, I copied most of that command from Renchap's post, and didn't double check everything :)
